src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -71,6 +70,20 @@ public ReportBuilder(List<String> jsonFiles, Configuration configuration) {
         reportParser = new ReportParser(configuration);
     }
 
+    public ReportBuilder(List<String> jsonFiles, Configuration configuration, ReportParser reportParser) {
```

Comment:
having getter and setter you don't need to add another constructor

Reply:
Indeed! it makes sense. 
Let me remove the constructor.

Reply:
I have updated the description too.